README2.md

```diff
@@ -0,0 +1,51 @@
+STRAIGHT UP
```

Comment:
Generally install instructions need to go at the top with what I need to do to install and run the web app successfully.